### ISSUE TITLE
Bots using CommonJS requiring the package are broken

### ISSUE THREAD

**Issue by Choooks22** [discordjs/voice#231]
### Issue description

#229 broke CommonJS `require`.

```
/project/node_modules/@discordjs/voice/dist/index.js:1
import path from "path";
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

### Code sample

```typescript
require('@discordjs/voice');
```

### @discordjs/voice version

0.7.3

### Node.js version

16.13.0

### Operating system

Linux

### Priority this issue should have

High (immediate attention needed)

**Comment by ZullsXakin** [comment]
I have the same thing...
![image](https://user-images.githubusercontent.com/76924369/141407745-e179ccc1-2136-44a8-abd7-346fa897f19f.png)

**Comment by iCrawl** [comment]
This was rather annoying to fix without writing a lot of code, I hope this is finally fixed in ~~0.7.4~~ 0.7.5 now 👍
